db/migrate/20210412172435_add_appointment_reminder_to_communication.rb

```diff
@@ -0,0 +1,5 @@
+class AddAppointmentReminderToCommunication < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :communications, :appointment_reminder, type: :uuid, foreign_key: true
```

Comment:
Theoretically this should be null false but that would invalidate all existing communications.